facade-app/src/test/Utils.test.tsx

```diff
@@ -0,0 +1,132 @@
+import {
```

Comment:
the build is failing, can you please fix that.

Reply:
Working on it.

Reply:
@preetkaran20 I am not sure what's the issue. Can anyone help me with this?

Reply:
Error is:
```
FAIL src/test/Utils.test.tsx (10.173 s)
  ● getResource() › should call the callback function with the obj when isJson is true

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      20 |
      21 | describe("getResource()", () => {
    > 22 |   it("should call the callback function with the obj when isJson is true", (done) => {
         |   ^
      23 |     //@ts-ignore
      24 |     const fetchMock = jest.spyOn(global, "fetch").mockResolvedValueOnce({
      25 |       ok: true,

      at src/test/Utils.test.tsx:22:3
      at Object.<anonymous> (src/test/Utils.test.tsx:21:1)

  ● getResource() › should call the callback function with the text when isJson is true

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      43 |   });
      44 |
    > 45 |   it("should call the callback function with the text when isJson is true", (done) => {
         |   ^
      46 |     //@ts-ignore
      47 |     const fetchMock = jest.spyOn(global, "fetch").mockRejectedValueOnce({
      48 |       ok: false,

      at src/test/Utils.test.tsx:45:3
      at Object.<anonymous> (src/test/Utils.test.tsx:21:1)
```

Reply:
is this test working in your local? can you please compare versions which you are using locally. The github action which is running is present at: https://github.com/SasanLabs/VulnerableApp-facade/blob/main/.github/workflows/Compile.yml